code/lib/core-events/src/errors/preview-errors.ts

```diff
@@ -238,6 +238,63 @@ export class StoryStoreAccessedBeforeInitializationError extends StorybookError
   }
 }
 
+export class MountMustBeDestructured extends StorybookError {
+  readonly category = Category.PREVIEW_API;
+
+  readonly code = 12;
+
+  constructor(public data: { playFunction: string }) {
+    super();
+  }
+
+  template() {
+    return dedent`
+    To use mount in the play function, you must use object destructuring, e.g. play: ({ mount }) => {}.
+    
+    Instead received:
+    ${this.data.playFunction}
+    `;
+  }
+}
+
+export class TestingLibraryMustBeConfigured extends StorybookError {
+  readonly category = Category.PREVIEW_API;
+
+  readonly code = 13;
+
+  template() {
+    return dedent`
+    You must configure testingLibraryRender to use play in portable stories.
+    
+    import { render } from '@testing-library/[renderer]';
+    
+    setProjectAnnotations({
+      testingLibraryRender: render,
+    });
+    
+    For other testing renderers, you can configure renderToCanvas:
+    
+    import { render } from 'your-renderer';
+    
+    setProjectAnnotations({
+      renderToCanvas: ({ storyFn }) => {
+        const Story = storyFn();
+        
+        // Svelte
+        render(Story.Component, Story.props);
+        
+        // Vue
+        render(Story);
+        
+        // or for React
+        render(<Story/>);
+      },
+    });
+    
+    `;
+  }
```

Comment:
@kylegach @shilman and @yannbf Do we want to make a dedicated docs page for this?

Reply:
Definitely! I stumbled upon this while testing a canary and it wasn't super clear. Having a documentation page would be really helpful.

Reply:
Will make a new PR for the docs!